src/LiveComponent/src/Test/TestLiveComponent.php

```diff
@@ -205,4 +212,19 @@ private function client(): KernelBrowser
 
         return $this->client;
     }
+
+    private function flattenFormValues(array $values, string $prefix = ''): array
+    {
+        $result = [];
+
+        foreach ($values as $key => $value) {
+            if (\is_array($value)) {
+                $result += $this->flattenFormValues($value, $prefix.$key.'.');
```

Comment:
No difference array_is_list / map ? (geniune question)

Does this work with fields accepting multiple values ?

Reply:
Don't know how to use `array_is_list` here. With `array_map` I can't handle keys.

Yes, I was a little undecided to not add a test for that method. At first I've thinked to add a public static method in `Symfony\UX\LiveComponent\Util\LiveFormUtility` which could be easily tested. But I don't know if it make sence as it will be created only for test purpose.

But this will work. 
```php
    public function testFlatten(): void
    {
        $data = [
            'foo' => ['bar' => 'baz'],
            'collection' => ['one', 'two', 'three'],
            'collection_with_sub' => [
                'sub_one' => ['child_one', 'child_two'],
                'sub_two' => ['child_three', 'key' => 'child_four'],
            ],
        ];

        $this->assertSame([
            'foo.bar' => 'baz',
            'collection.0' => 'one',
            'collection.1' => 'two',
            'collection.2' => 'three',
            'collection_with_sub.sub_one.0' => 'child_one',
            'collection_with_sub.sub_one.1' => 'child_two',
            'collection_with_sub.sub_two.0' => 'child_three',
            'collection_with_sub.sub_two.key' => 'child_four',
        ], TestLiveComponent::flattenFormValues($data));
    }
```

Reply:
Good for me then, thank you for taking time to answer :)